subjects/java/checkpoints/steady-sequence/README.md

```diff
@@ -0,0 +1,53 @@
+## Steady Sequence
+
+### Instructions
+
+Create a class named SteadySequence that includes a method to determine the length of the longest consecutive sequence in an unsorted array of integers. The "longest consecutive sequence" refers to the sequence with the longest stretch of elements that increase strictly by 1. The elements of this sequence may appear in any order within the array.
```

Comment:
Minor thing `SteadySequence` is not formatted

Reply:
done